Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -445,7 +447,7 @@ export class Job {
                     thisJob.stopWork(0, JobState.Finishing);
                 }
             }
-        }).auth(thisJob.queue.TaskOptions.username, thisJob.queue.TaskOptions.password, true)
+        }).auth(thisJob.queue.TaskOptions.username, thisJob.queue.TaskOptions.password, false)
```

Comment:
This parameter is the 'sendImmeadiately' flag. When set to false it forces the request to retry with the proper authentication headers. See this doc for reference: https://www.npmjs.com/package/request#http-authentication)

Reply:
Comment in code, plz!

Reply:
Done!